deployment/https/env-docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+version: "3.9"
+services:
+  nginx:
+    image: nginx:latest
+    restart: always
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+      - ./data/certbot/conf:/etc/letsencrypt
+      - ./data/certbot/www:/var/www/certbot
+      - ./conf.d:/etc/nginx/conf.d
+      - ./content:/var/www/html
+    ports:
+      - 80:80
+      - 443:443
+    networks: # Networks to join (Services on the same network can communicate with each other using their name)
+      - backend
+
+  certbot:
+    image: certbot/certbot:latest
+    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
+    command: "/bin/sh -c 'while :; do sleep 6h & wait $${!}; nginx -s reload; done & nginx -g \"daemon off;\"'"
+    volumes:
+      - ./data/certbot/conf:/etc/letsencrypt
+      - ./certbot/logs:/var/log/letsencrypt
+      - ./data/certbot/www:/var/www/certbot
+
+  db-keycloak:
+    image: mariadb:latest
+    restart: on-failure
+    environment:
+      MYSQL_DATABASE: 'keycloak'
+      MYSQL_USER: 'keycloak'
+      MYSQL_PASSWORD: ${DB_PW_KEYCLOAK}
+      MYSQL_ALLOW_EMPTY_PASSWORD: 'no'
+      MYSQL_ROOT_PASSWORD: ${DB_PW_KC_ROOT}
+      MYSQL_TCP_PORT: 3307
+    healthcheck:
+      test: mysqladmin ping -h 127.0.0.1 -P 3307 -u $$MYSQL_USER --password=$$MYSQL_PASSWORD
+      interval: 5s
+      timeout: 60s
+      retries: 30
+    volumes:
+      - /mnt/lj-mariadb/keycloak:/var/lib/mysql
+    networks: 
+      - backend
+
+  keycloak:
+    image: jboss/keycloak
+    volumes:
+      - ./../keycloak/imports:/opt/jboss/keycloak/imports
+    depends_on:
+      db-keycloak:
+        condition: service_healthy
+    restart: on-failure
+    environment:
+      KEYCLOAK_IMPORT: /opt/jboss/keycloak/imports/realm.json
+      KEYCLOAK_USER: admin
```

Comment:
This should be an external variable

Reply:
Because admin is not a secure admin username ;)

Reply:
done